src/app/app.component.html

```diff
@@ -1,2 +1,2 @@
 <h1>NgWig Demo</h1>
-<ng2wig></ng2wig>
+<ng2wig plugins="demo"></ng2wig>
```

Comment:
need I bit more time to dive into it. 
created a question on SO - http://stackoverflow.com/questions/42995994/angular-plugins-for-3rd-party-packages - it would be nice to get some feedback